src/lib/components/ChatMessage.svelte

```diff
@@ -455,6 +591,15 @@
             <emoji-picker bind:this={emojiToolbarPicker}></emoji-picker>
           </Popover.Content>
         </Popover.Root>
+        {#if mayEdit}
+          <Toolbar.Button
+            onclick={() => startEditing()}
+            class="btn btn-ghost btn-square"
+          >
+            <Icon icon="tabler:edit" />
+          </Toolbar.Button>
+        {/if}
```

Comment:
After clicking here the popover should hide. Right now it still shows behind the chat box in an awkward way.